tasks/linter.py

```diff
@@ -783,3 +784,42 @@ def gitlab_ci_jobs_owners(_, diff_file=None, config_file=None, path_jobowners='.
         )
     else:
         print(f'{color_message("Success", Color.GREEN)}: All jobs have owners defined in {path_jobowners}')
+
+
+@task
+def gitlab_ci_jobs_codeowners(ctx, path_codeowners='.github/CODEOWNERS', all_files=False):
+    """
+    Verifies that added / modified job files are defined within CODEOWNERS.
+    """
+
+    if all_files:
+        modified = glob('.gitlab/**/*.yml', recursive=True)
+    else:
+        modified = get_file_modifications(ctx, added=True, modified=True, only_names=True)
+        modified = [path for path in modified if path.endswith('.yml')]
+
+    with open(path_codeowners) as f:
+        lines = f.readlines()
```

Comment:
```suggestion
        parsed_owners = f.readlines()
```
You could also use [this](https://github.com/DataDog/datadog-agent/blob/main/tasks/libs/owners/parsing.py#L7)

Reply:
As some preprocessing is made and only using `CodeOwners(<content>)` from this function is necessary, adding the preprocessing within this function is less readable IMO